lib/node_modules/@stdlib/regexp/to-json/docs/repl.txt

```diff
@@ -0,0 +1,27 @@
+
+{{alias}}( regexp )
+    Returns a JSON representation of a regular expression.
+
+    The returned object has the following properties:
+
+    - type: value type. The assigned value is always 'RegExp'.
+    - pattern: regular expression pattern.
+    - flags: regular expression flags.
+
+    Parameters
+    ----------
+    regexp: RegExp
+        Regular expression to serialize.
+
+    Returns
+    -------
+    out: Object
+        JSON representation.
+
+    Examples
+    --------
+    > var json = {{alias}}( /ab+c/ )
+    <Object>
+
+    See Also
+    --------
```

Comment:
```suggestion
    --------
    
```

I think we're missing a blank line here. Should have 2 line breaks after the See Also section.